release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -291,6 +291,35 @@ Currently, it is not supported to use Active Directory Federation Services with
 
 For the full list of OIDC providers, see xref:../authentication/identity_providers/configuring-oidc-identity-provider.adoc#identity-provider-oidc-supported_configuring-oidc-identity-provider[Supported OIDC providers].
 
+[id="ocp-4-11-auth-pod-security-admission"]
+==== Pod security admission
+
+link:https://kubernetes.io/docs/concepts/security/pod-security-admission/[Pod security admission] is now enabled on {product-title}. 
+
+Clusters support pod security admission, as well as SCC admission, for pod admission. Pod security admission runs globally with "privileged" enforcement and "restricted" audit logging and API warnings.
```

Comment:
```suggestion
Pod admission is therefore enforced by both the Pod Security and the SCC admissions. Pod security admission runs globally with "privileged" enforcement and "restricted" audit logging and API warnings.
```
- to maybe go a bit easy on the repetition of "admission" here

Reply:
```suggestion
Pod admission is enforced by both pod security and the SCC admissions. Pod security admission runs globally with "privileged" enforcement and "restricted" audit logging and API warnings.
```

Reply:
again, not a native speaker, but I assume that you'd have to use `the` with both the subjects, or neither - `pod security and SCC admissions`

Reply:
Fixed!